mgmtsystem_audit/models/mgmtsystem_audit.py

```diff
@@ -20,163 +20,224 @@
-            'mgmtsystem.verification.line',
-            'audit_id',
-            'Verification List',
-        ),
-        'user_id': fields.many2one('res.users', 'Audit Manager'),
-        'auditor_user_ids': fields.many2many(
-            'res.users',
-            'mgmtsystem_auditor_user_rel',
-            'user_id',
-            'mgmtsystem_audit_id',
-            'Auditors',
-        ),
-        'auditee_user_ids': fields.many2many(
-            'res.users',
-            'mgmtsystem_auditee_user_rel',
-            'user_id',
-            'mgmtsystem_audit_id',
-            'Auditees',
-        ),
-        'strong_points': fields.text('Strong Points'),
-        'to_improve_points': fields.text('Points To Improve'),
-        'imp_opp_ids': fields.many2many(
-            'mgmtsystem.action',
-            'mgmtsystem_audit_imp_opp_rel',
-            'mgmtsystem_action_id',
-            'mgmtsystem_audit_id',
-            'Improvement Opportunities',
-        ),
-        'nonconformity_ids': fields.many2many(
-            'mgmtsystem.nonconformity',
-            string='Nonconformities',
-        ),
-        'state': fields.selection(
-            [
-                ('open', 'Open'),
-                ('done', 'Closed'),
-            ],
-            'State',
-        ),
-        'system_id': fields.many2one('mgmtsystem.system', 'System'),
-        'company_id': fields.many2one('res.company', 'Company')
-    }
-
-    _defaults = {
-        'company_id': (
-            lambda self, cr, uid, c:
-            self.pool.get('res.users').browse(cr, uid, uid, c).company_id.id),
-        'reference': 'NEW',
-        'state': 'open'
-    }
-
-    def create(self, cr, uid, vals, context=None):
+    name = fields.Char('Name', size=50)
+
+    reference = fields.Char(
+        'Reference',
+        size=64,
+        required=True,
+        readonly=True,
+        default='NEW'
+    )
+    date = fields.Datetime('Date')
+    line_ids = fields.One2many(
+        'mgmtsystem.verification.line',
+        'audit_id',
+        'Verification List',
+    )
+    quantity = fields.Integer('Number of audits', readonly=True, default=1)
+    number_of_nonconformities = fields.Integer(
+        'Number of nonconformities',
+        compute='_number_of_nonconformity', readonly=True)
+    number_of_questions_in_verification_list = fields.Integer(
+        'Number of questions in verification list',
+        compute='_number_of_questions_in_verification_list', readonly=True)
+    number_of_improvements_opportunity = fields.Integer(
+        'Number of improvements Opportunities',
+        compute='_number_of_improvement_opportunity', readonly=True)
+
+    closing_date = fields.Datetime('Closing Date', readonly=True)
+
+    age = fields.Integer('Age', readonly=True, compute='_get_age')
+
+    def _get_age(self):
+        """Get audit current age."""
+        return (
+            datetime.now() - datetime.strptime(self.date, "%Y-%m-%d %H:%M:%S")
```

Comment:
use `fields.Datetime.now()` and 
use `fields.Datetime.from_string(<date_field>)`